pkgs/cosmic-session/package.nix

```
@@ -50,7 +50,7 @@ rustPlatform.buildRustPackage {
     "cosmic"
   ];
 
-  env.XDP_COSMIC = lib.getExe xdg-desktop-portal-cosmic;
+  env.XDP_COSMIC = "${xdg-desktop-portal-cosmic}/libexec/xdg-desktop-portal-cosmic";
```

```suggestion
  env.XDP_COSMIC = "${lib.getLib xdg-desktop-portal-cosmic}/libexec/xdg-desktop-portal-cosmic";
```

Might be good to select output, but it admittedly doesn't matter here (just in a hypothetical future if outputs were ever split, it's generally a best practice to select relevant output whether it's already split or not)

I'm fine to merge either way :)